lte/gateway/c/oai/tasks/mme_app/mme_app_purge_ue.c

```diff
@@ -70,7 +70,8 @@ int mme_app_send_s6a_purge_ue_req(mme_app_desc_t *mme_app_desc_p,
     OAILOG_WARNING(
       LOG_MME_APP,
       "Failed to allocate memory for S6A_PURGE_UE_REQ for imsi " IMSI_64_FMT
-      "\n", imsi);
+      "\n",
```

Comment:
Reword message to fit in one line, such as "Failed to allocate memory for S6A_PURGE_UE_REQ, imsi:" IMSI_64_FMT "\n"

Reply:
Fixed in the updated commit.